Rakefile

```diff
@@ -68,11 +73,13 @@ task ci: [:lint, :analyze, :test]
 
 # task release: :test do
 task release: :ci do
-  sh "npm ci"
   abort red 'Must be on master branch' unless current_branch == 'master'
   abort red 'Must have push access to Backpack on CocoaPods trunk' unless has_trunk_push
   abort red 'Git branch is not up to date please pull' unless branch_up_to_date
 
+  sh "npm ci"
```

Comment:
I moved this as we might as well get the fast checks above done and passed before we make the person releasing wait for `npm ci`